docker-compose.server.yml

```diff
@@ -0,0 +1,17 @@
+version: "3.6"
+
+services:
+  web:
+    build: .
+    image: "cornellappdev/pear:${IMAGE_TAG}"
+    env_file: node.env
```

Comment:
can we point to `.envrc`?

Reply:
no, we can't -- this is a server docker compose file